test/get_os_log.py

```diff
@@ -17,7 +18,10 @@ def set_passcode(harness: Harness, flag: bool):
     '''
     on exit -> restore PhoneModeLock
     '''
-    PhoneModeLock(flag).run(harness)
+    try:
+        PhoneModeLock(flag).run(harness)
+    except:
+        pass
```

Comment:
this is plain wrong.

Reply:
This is a poor-man handler of the case when a passcode is not enabled on the phone. Sure, there should be some information about why we ignored the exception.

Reply:
Thank you. Fixed.